src/actors/objects/mod.rs

```diff
@@ -8,6 +8,7 @@ use actix::{Actor, Addr, Context, Handler, Message, ResponseFuture};
 use bytes::Bytes;
 use failure::{Fail, ResultExt};
 
+use ::sentry::integrations::failure::capture_fail;
```

Comment:
_nit_: Remove the leading `::`

Reply:
as discussed irl, not possible due to `objects::sentry` and us being in `mod.rs` rn